helm/hpcc/values.schema.json

```diff
@@ -2946,6 +2967,131 @@
       "description": "A list of planes that will take precedence to read from if logical files reside on mutiple planes",
       "type": "array",
       "items": { "type": "string" }
+    },
+    "hpa" : {
+      "type": "object",
+      "description": "Horizontal Pod Autoscaler parameters",
+      "properties": {
+        "minReplicas": {
+          "description": "Lower limit for number of replicas which the autoscaler can scale down. It defaults to 1 pod.",
+          "type": "integer"
+        },
+        "maxReplicas": {
+          "description": "Upper limit for number of replicas which the autoscaler can scale up. It cannot be less that minReplicas",
+          "type": "integer"
+        },
+        "behavior": {
+          "type": "object",
+          "description": "Configures the scaling behavior of the target in both Up and Down directions",
+          "properties": {
+            "scaleUp": {
+              "type": "object",
+              "$ref": "#/definitions/hpaScalingRules"
+            },
+            "scaleDown": {
+              "type": "object",
+              "$ref": "#/definitions/hpaScalingRules"
+            }
+          }
+        },
+        "metrics": {
+          "type": "array",
+          "description": "Specifies variables used to calculate the desired replica count. The desired replica count is calculated multiplying the ratio between the target value and the current value by the current number of pods. ",
+          "items": { "$ref": "#/definitions/hpaMetric" }
+        }
+      },
+      "required": [ "maxReplicas", "metrics" ]
+    },
+    "hpaScalingRules": {
+      "type": "object",
+      "properties": {
+        "policies" : {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "type": {
+                "type": "string",
+                "description": "Type is used to specify the scaling policy"
+              },
+              "value": {
+                "type": "integer",
+                "description": "Value contains the amount of change which is permitted by the policy. It must be greater than zero"
+              },
+              "periodSeconds": {
+                "type": "integer",
+                "description": "PeriodSeconds specifies the window of time for which the policy should hold true. PeriodSeconds must be greater than zero and less than or equal to 1800 (30 min)."
```

Comment:
minor: casing. "Value" and "Type" also, vs 'selectPolicy' or 'selector' in descriptions below.

I suspect if going to mention the option in the description the casing should match, but it is probably best to describe without what it is without re-mentioning it.